src/azure-cli-core/tests/test_profile.py

```diff
@@ -451,6 +451,38 @@ def test_find_subscriptions_thru_username_password(self, mock_auth_context):
         mock_auth_context.acquire_token.assert_called_once_with(
             mgmt_resource, self.user1, mock.ANY)
 
+    @mock.patch('adal.AuthenticationContext.acquire_token_with_username_password', autospec=True)
+    @mock.patch('adal.AuthenticationContext.acquire_token', autospec=True)
+    @mock.patch('azure.cli.core._profile.CLOUD', autospec=True)
+    def test_find_subscriptions_thru_username_password_azure_stack(self, mock_get_cloud, mock_acquire_token,
```

Comment:
This test is for ADFS which *may* be available on other clouds right?
I tihnk `azure_stack` should be removed from the name of the test and maybe `adfs` in its place.